content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.OI
+  - ib-19.3.Datatype-Primitief-Integer.OI
+---
+
+## Wat is een Integer?
+Een Integer is een datatype waar een positief of negatief getal in past. Een programmeur kan dit datatype aan een variabel geven en hiermee getallen bewaren, bewerken en uitlezen.
+
+> [!TIP] Casus
+> Stel je hebt een evenement met toegangspoortjes georganiseerd. Je wilt graag bijhouden hoeveel mensen het evenement betreden.  
+
+## Hoe zit een Integer in elkaar?
+De Integer wordt in C# ingezet door bij het initialiseren `int` voor het variabel-naam te zetten. Het variabel krijgt dan het datatype Integer.
+
+### Codevoorbeeld
+```C#
+int variabelNaam;
+```
+
+Hier kan dan een nummer aan toegevoegd worden door het variabel een waarde toe te wijzen.
+
+### Codevoorbeeld
+```C#
+int leeftijd;
+leeftijd = 17;
+```
+
+## Hoe gebruik je een Integer?
+Een Integer is een gegevenstype dat hele getallen opslaat, zoals 0, -1, 42 of 100. Er mogen in een Integer dus geen kommagetallen staan. Als je wel een komma getal wil opslaan, kan je gebruik maken van de bijvoorbeeld de -=DOUBLE=- of -=FLOAT=-.
```

Comment:
Opmerking: kommagetal schrijf je aan elkaar. De 'de' voor 'bijvoorbeeld' is denk ik onnodig.